resources/lib/ui/embed_extractor.py

```diff
@@ -253,7 +268,7 @@ def f(url, content):
 
 __register_extractor("http://embed.novamov.com/", __extract_swf_player)
 
-__register_extractor("https://openload.co/embed/", __extract_openload)
+#__register_extractor("https://openload.co/embed/", __extract_openload)
```

Comment:
i know how to pass openload too just didn't had the time for that,
we can talk if you are interested 👍